NotificationApi/NotificationApi/Controllers/NotificationController.cs

```diff
@@ -7,90 +7,56 @@
-                Id = template.Id,
-                NotificationType = (Contract.NotificationType)template.NotificationType,
-                NotifyTemplateId = template.NotifyTemplateId,
-                Parameters = template.Parameters
-            });
-        }
-
-        [HttpPost]
-        [OpenApiOperation("CreateNewNotification")]
-        [ProducesResponseType((int)HttpStatusCode.OK)]
-        [ProducesResponseType(typeof(string), (int) HttpStatusCode.InternalServerError)]
-        [ProducesResponseType(typeof(ValidationProblemDetails), (int)HttpStatusCode.BadRequest)]
-        [Obsolete("Please use the journey specific routes")]
-        public async Task<IActionResult> CreateNewNotificationAsync(AddNotificationRequest request)
+        
+        return Ok(new NotificationTemplateResponse
         {
-            var parameters = JsonConvert.SerializeObject(request.Parameters);
-            var emailNotifications = await _queryHandler.Handle<GetEmailNotificationQuery, IList<EmailNotification>>(
-                new GetEmailNotificationQuery(request.HearingId, request.ParticipantId,
-                    (NotificationType)request.NotificationType, request.ContactEmail));
-            var emailNotification = emailNotifications.SingleOrDefault(x => x.Parameters == parameters);
-
-            if (emailNotification == null)
-            {
-                var notification = new CreateEmailNotificationCommand((NotificationType)request.NotificationType,
-                    request.ContactEmail, request.ParticipantId, request.HearingId, parameters);
-                await _createNotificationService.CreateEmailNotificationAsync(notification, request.Parameters);
-            }
-
-            return Ok();
-        }
-
-        /// <summary>
-        /// Process callbacks from Gov Notify API
-        /// </summary>
-        /// <returns></returns>
-        [HttpPost("callback")]
-        [OpenApiOperation("HandleCallback")]
-        [Authorize(AuthenticationSchemes = "Callback")]
-        [ProducesResponseType((int) HttpStatusCode.OK)]
-        [ProducesResponseType(typeof(string), (int) HttpStatusCode.InternalServerError)]
-        [ProducesResponseType(typeof(ValidationProblemDetails), (int)HttpStatusCode.BadRequest)]
-        public async Task<IActionResult> HandleCallbackAsync(NotificationCallbackRequest notificationCallbackRequest)
+            Id = template.Id,
+            NotificationType = (Contract.NotificationType)template.NotificationType,
+            NotifyTemplateId = template.NotifyTemplateId,
+            Parameters = template.Parameters
+        });
+    }
+    
+    [HttpPost]
+    [OpenApiOperation("CreateNewNotification")]
+    [ProducesResponseType((int)HttpStatusCode.OK)]
+    [ProducesResponseType(typeof(string), (int) HttpStatusCode.InternalServerError)]
+    [ProducesResponseType(typeof(ValidationProblemDetails), (int)HttpStatusCode.BadRequest)]
+    [Obsolete("Please use the journey specific routes, used to seed tests")]
```

Comment:
just wondering if we can remove this endpoint now. BQS no longer calls it, and that is the only client as far as I'm aware

Reply:
It's use be the NotificationAPI integration tests to seed data I believe

Reply:
hmm, ok. Might be worth seeing if we can remove it as part of future tech debt, if not part of this one. Might need to rejig the seeding tests to use the individual journey endpoints

Reply:
Yeah good idea, will spin up a ticket for it